src/test/resources/application-test.yml

```diff
@@ -27,3 +27,4 @@ api:
     url:
       hmpps-auth: http://localhost:8090/auth
       locations-inside-prison: http://localhost:8091
+      prisoner-search: http://localhost:8092
```

Comment:
Are we going with incremental ports for test, so
auth - 8090
locations - 8091
prisoner-search - 8092
xxxx - 8093
yyyy - 8094

Reply:
That's what I thought to keep things simple.  Open to other suggestions if you have them.